src/VisualStudio/CSharp/Test/Interactive/Commands/InteractiveCommandHandlerTestStub.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Editor.Interactive;
+using Microsoft.VisualStudio.InteractiveWindow;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Operations;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Interactive.Commands
+{
+    internal class InteractiveCommandHandlerTestStub : InteractiveCommandHandler
```

Comment:
I assume this is just following a pattern, but I don't find the name "TestStub" very clear.